src/mumble/main.cpp

```diff
@@ -162,14 +162,16 @@ int main(int argc, char **argv) {
 					"                Allow multiple instances of the client to be started.\n"
 					"  -n, --noidentity\n"
 					"                Suppress loading of identity files (i.e., certificates.)\n"
-					"  -jn, --jackname\n"
+					"  -jn, --jackname <arg>\n"
 					"                Set custom Jack client name.\n"
 					"  --license\n"
 					"                Show the Mumble license.\n"
 					"  --authors\n"
 					"                Show the Mumble authors.\n"
 					"  --third-party-licenses\n"
 					"                Show licenses for third-party software used by Mumble.\n"
+					"  --posttitle <arg>\n"
```

Comment:
The same character for two subsequent times may be confusing.

```suggestion
					"  --post-title <arg>\n"
```

Reply:
True